AGInputControls/Source/FloatingLabelTextField.swift

```diff
@@ -211,8 +229,20 @@ open class FloatingLabelTextField : FormattingTextField {
     }
     
     private func configurePlaceholder() {
-        guard placeholder != nil else { return }
-        placeholderLabel.text = placeholder
+        guard let placeholder else { return }
```

Comment:
by returning here line 246 won't be called. lets check if we need that line at all